src/Configurations/test/Responses.test.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { waitFor, screen } from '@testing-library/react';
+import user from '@testing-library/user-event';
+
+import * as components from '@folio/stripes-acq-components';
+
+import { mockKy, server } from '../../test/net';
+import {
+  mockedConfigurations, mockedProviders,
+  mockedSingleConfiguration,
+  renderConfigurations,
+} from './setup';
+
+
+jest.mock('react-virtualized-auto-sizer', () => ({ children }) => children({ width: 1920, height: 1080 }));
+
+jest.mock('@folio/stripes-components/lib/Icon', () => props => (
+  <span data-testid={props['data-testid']}>
+    <svg />
+    {props.children && <span>{props.children}</span>}
+  </span>
+));
+
+jest.mock('@folio/stripes-acq-components', () => ({
+  ...jest.requireActual('@folio/stripes-acq-components'),
+  useShowCallout: jest.fn(),
+}));
+
+jest.mock('@folio/stripes/core', () => ({
+  ...jest.requireActual('@folio/stripes/core'),
+  useOkapiKy: () => mockKy,
+  useStripes: () => ({
+    hasPerm: jest.fn().mockReturnValue(true),
+  }),
+  IfPermission: props => <>{props.children}</>,
+}));
+
+
+const renderSingleConfiguration = async () => {
+  renderConfigurations();
+
+  const cell = await screen.findByRole('gridcell', { name: 'RS1' });
+  user.click(cell);
+};
```

Comment:
```suggestion
const renderSingleConfiguration = async () => {
  renderConfigurations();

  const cell = await screen.findByRole('gridcell', { name: 'RS1' });
  user.click(cell);
  await screen.findByRole('heading', { name: /details/ });
};
```
Did you see in the console this warning in red:

```
When testing, code that causes React state updates should be wrapped into act(...)
```

That's because in this async function we made a click, which caused some state change, but we did not wait for any state change before going on with the test.
In other words, `await renderSingleConfiguration()` did not wait for click results.
Now it does.

See the [article](https://kentcdodds.com/blog/fix-the-not-wrapped-in-act-warning/) on this matter